docs/team/ernestcuong.md

```diff
@@ -5,13 +5,42 @@ title: Cuong's Project Portfolio Page
 
 ### Project: TutorAid
 
-TutorAid is a desktop application used for helping tech-savvy private tutors, who have busy schedules / many students, to keep track of each students' details. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
+TutorAid is a desktop application used for helping tech-savvy private tutors, who have busy schedules / many students, to keep track of the details of all their students and lessons. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
 
 Given below are my contributions to the project.
 
-* **Documentation**:
-  * User Guide:
-    * Added documentation for the features `add -p` and `del -p` [\#72]()
-  * Developer Guide:
-    * Target user profile, value proposition, user stories, use cases, non-functional requirements, glossary
+* **Code contributed**:
+  * My contributions to the TutorAid code base can be found [here](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=W16&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17&tabOpen=true&tabType=authorship&tabAuthor=ErnestCuong&tabRepo=AY2122S1-CS2103T-W16-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false).
+
+* **New features implemented**:
+  * Implemented a `progress` feature for student for v1.2.
+    * Relevant PR(s): [#64](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/64/files)
+    * Implemented the `add -p` and `del -p` commands to add and delete progress from a student.
+    * Worked closely with the `logic` component to implement the commands and the parser for adding and deleting progress, as well as `model` to add a new `Progress` field to `Student`
```

Comment:
Maybe this should be `Logic` and `Model` (capitalise)

Reply:
Thanks a lot